Weekly Reflection 13

HadesGeneral opened the pull request:
Spell checked it all and so I can make a pull request to link the new issue.

HadesGeneral committed:
Updated File

Spell checked it all and so I can make a pull request to link the new issue.

belainehi commented on Reflections/Week_13_Reflection.md:
```diff
@@ -1,22 +1,22 @@
 1. What have you accomplished this week? (Please list each accomplishment, and explain) 
-  - I was able to conncect the Linear Actuator to the currently built system. (D4.5 shows this.)
-  - I was able to get it to work together with only a few hickups, since they were built sepertly, so new parts needed to be intergrated or moved around. (D4.5 shows this.)
+  - I was able to connect the Linear Actuator to the currently built system. (D4.5 shows this.)
```
It's great that you connected everything together. Does the remote control app work the same way?

belainehi commented on Reflections/Week_13_Reflection.md:
```diff
@@ -1,22 +1,22 @@
   - I was able to test and show off the system working in it's entirety. (D4.5 shows this.)
 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
-  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so i moved it to under the Linear Actuator, and right beside the Lego Brain. I achived this by transfering the rotational movement of the motor with a axle and gear setup. (D4.5 shows this.)
+  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so, I moved it to under the Linear Actuator, and right beside the Lego Brain. I achieved this by transferring the rotational movement of the motor with an axle and gear setup. (D4.5 shows this.)
```
I watched the video and it's great that your system is able to change directions along with the weight. In you D4.5 can you specify how much weight you tested your system with? And how much the whole system weighs with the new changes?

belainehi commented on Reflections/Week_13_Reflection.md:
```diff
@@ -1,22 +1,22 @@
-    - Measurable: If the weight doesn't fall off or it doesn't move as easily loying down, then I have achived the goal.
+    - Specific: The platform on the linear actuator currently cannot support the weight correctly. It needs duct tape as extra support, and even then, it is not as structurally sound. So, making it where the weight can lay down without falling off when using the system will be the goal.
+    - Measurable: If the weight does not fall off or it does not move as easily lying down, then I have achieved the goal.
     - Achievable: This is kind achievable. The hard part is going to be making sure I don't change the setup to much so that it still works as intended. 
```
You must give an estimate of how much time you plan to spend working on this, and when you expect results for all your S.M.A.R.T. goals.

belainehi commented on Reflections/Week_13_Reflection.md:
```diff
@@ -1,22 +1,22 @@
+    - Measurable: If the button is pressed and it stops functions for a moment, then makes the skateboard back up then this shows I have done it. 
+    - Achievable: This is sort of because the coding is semi-unfamiliar so coding this will be hard. I also do not want to break the system I have built so far, so I am wary for it all so far.
 4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
   - Current system
```
You should list out your components as well as anything you'll use for testing.

HadesGeneral committed:
Updated File

Added weight amounts

HadesGeneral committed:
Updated File

Added time estimates for goals

HadesGeneral committed:
Updated File

Added components in resources and time estimates for goals

HadesGeneral committed:
Updated file

Added system weight

HadesGeneral committed:
Added Photo

Added a screenshot of the code for the backing up button

HadesGeneral committed:
Uploaded Button Sensor Photos

Added all the rest of the photos related to the button sensor

HadesGeneral committed:
Uploaded Code Screen Shot

Annotated the Code screen shot for the backup program

HadesGeneral committed:
Updated File file to have Button Goal

Added all information and photos needed to show off the button goal from Weekly reflection 13

HadesGeneral committed:
Added photos

Added photos dealing with the weight platform goal for Weekly Reflection 13

HadesGeneral committed:
Updated D4.5 with weight Goal

Added the details and photos for the Weight platform change goal in Weekly Reflection 13

HadesGeneral committed:
Updated File with Youtube link

added the link and description of the youtube video showing off the button sensor testing.

jazzij commented:
Deliverable 4 document should only contain the documentation for your Low Fidelity prototype. Nothing submitted in Spring A will be graded again in Spring B. Anything more recent should be in its own implementation branch and document, organized in a reasonable manner, such as Week by Week process or Component by Component (Showing evolution of component development). Please rearrange your Semester B documentation accordingly

HadesGeneral committed:
Updated D4

Took out anything from session B and transferred it to D4.5

HadesGeneral committed:
Updated D4.5

Added Session B stuff from D4 at the top of this document.

belainehi commented:
For your testing, I recommend  Functional Testing (Both Unit and Integration) and End to End testing. Possible component groups to do a unit test over: 

Group 1: NXT Brain and Remote Control 
Group 1.1: NXT Brain, Skateboard and Remote Control
Group 2: Linear Actuator and Remote Control 
Group 3: Linear Actuator with weight and Remote Control
**(Add more components groups that make sense)**

It is expected that you will come up with component groups that make sense for your project and that you will document your testing for D5. You have already been testing as you've been building your project so it shouldn't be too hard.

HadesGeneral commented:
Can I merge this?

belainehi commented:
@HadesGeneral you can merge this. The Week_13_Reflection.md file will cause a merge conflict though, you might need to use command line. Can you come to lab on Saturday?

HadesGeneral commented:
@belainehi I don't know how to use the command line. I have been using strickly the browser. When is the lab, there's a good chance I can not make it?
